packages/vscode-ui5-language-assistant-bas-ext/scripts/copy.js

```diff
@@ -0,0 +1,32 @@
+const { copySync, readdirSync } = require("fs-extra");
+const { join } = require("path");
+
+const baseSrc = join(__dirname, "..", "..", "vscode-ui5-language-assistant");
+const baseDes = join(__dirname, "..");
+
+const vsixFiles = [];
+readdirSync(baseSrc).forEach((item) => {
+  if (item.endsWith(".vsix")) {
+    vsixFiles.push(item);
+    return item;
+  }
+});
+if (vsixFiles.length > 1) {
+  throw new Error(
+    `Detected more than one ".vsix" files. There should be only one ".vsix" file. Please cross check and try again.`,
+    { cause: vsixFiles }
+  );
+}
+const vsixFile = vsixFiles.pop();
+if (!vsixFile) {
```

Comment:
you can probably combine these two checks with vsixFiles.length === 1

But this is not mandatory...